### PR TITLE
Cleanup CODEOWNERS file

### PR DESCRIPTION
**Pull request by nashif**
fix path and owners...

### CODE REVIEW

**Review by himanshujha199640 — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -188,7 +188,7 @@ include/toolchain/                       @andrewboie @andyross
 include/zephyr.h                         @andrewboie @andyross
 kernel/                                  @andrewboie @andyross
 lib/gui/                                 @vanwinkeljan
-lib/posix/                               @ramakrishnapallala @nniranjhana @pfalcon
+lib/posix/                               @pfalcon
```
Comment (pfalcon) [inline CODEOWNERS]: Well, there're regular comments and even PRs from those guys, and I'm not comfortable to be there all alone ;-).

File: CODEOWNERS [hunk 2]
```diff
@@ -6,250 +6,250 @@
...
-boards/nios2/altera_max10/               @ramakrishnapallala
-boards/posix/                            @aescolar
-boards/riscv32/                          @kgugala @pgielda @nategraff-sifive
-boards/x86/                              @andrewboie @nashif
-boards/x86/arduino_101/                  @nashif
-boards/x86/galileo/                      @nashif
-boards/x86/quark_d2000_crb/              @nashif
-boards/x86/quark_se_c1000_devboard/      @nashif
-boards/xtensa/                           @andrewboie @ramakrishnapallala
+/.known-issues/                           @inakypg @nashif
```
Comment (pfalcon) [inline CODEOWNERS]: Could have named the commit "*prepend* / to all entries" or "use absolute paths for all entries".

File: CODEOWNERS [hunk 3]
```diff
@@ -192,24 +192,21 @@ lib/posix/                               @pfalcon
...
 samples/gui/                             @vanwinkeljan
 samples/net/                             @jukkar @tbursztyka @pfalcon
 samples/net/dns_resolve/                 @jukkar @tbursztyka @pfalcon
-samples/net/http_server/                 @jukkar @tbursztyka
 samples/net/lwm2m_client/                @mike-scott
-samples/net/mbedtls_sslclient/           @jukkar
 samples/net/mqtt_publisher/              @jukkar @tbursztyka
-samples/net/coap_client/                 @rveerama1
-samples/net/coap_server/                 @rveerama1
+samples/net/sockets/coap_client/         @rveerama1
```
Comment (pfalcon) [inline CODEOWNERS]: I guess, @mike-scott should be added to these, but I guess, he can do that in a separate PR.

**Review by aescolar — COMMENTED** [review]
Havent seen any problem, but I haven't gone thru all lines

File: CODEOWNERS [hunk 4]
```diff
@@ -6,254 +6,250 @@
...
+/boards/riscv32/                          @kgugala @pgielda @nategraff-sifive
+/boards/x86/                              @andrewboie @nashif
+/boards/x86/arduino_101/                  @nashif
+/boards/x86/galileo/                      @nashif
+/boards/x86/quark_d2000_crb/              @nashif
+/boards/x86/quark_se_c1000_devboard/      @nashif
+/boards/xtensa/                           @andrewboie @ramakrishnapallala
 # All cmake related files
-cmake/                                   @nashif @SebastianBoe
+/cmake/                                   @nashif @SebastianBoe
```
Comment (carlescufi) [inline CODEOWNERS]: Is there an indentation mismatch below?

File: CODEOWNERS [hunk 5]
```diff
@@ -6,254 +6,250 @@
...
+/tests/drivers/spi/                       @tbursztyka
+/tests/kernel/                            @andrewboie @andyross @nashif
+/tests/net/                               @jukkar @tbursztyka @pfalcon
+/tests/net/buf/                           @jukkar @jhedberg @tbursztyka @pfalcon
+/tests/net/lib/                           @jukkar @tbursztyka @pfalcon
+/tests/net/lib/http_header_fields/        @jukkar @tbursztyka
+/tests/net/lib/mqtt_packet/               @jukkar @tbursztyka
+/tests/net/lib/coap/                      @rveerama1
+/tests/net/socket/                        @jukkar @tbursztyka @pfalcon
+/tests/subsys/fs/                         @nashif @ramakrishnapallala
```
Comment (carlescufi) [inline CODEOWNERS]: line below also has an indentation issue

File: CODEOWNERS (hunk 4 above)
Comment (nashif) [reply CODEOWNERS]: no, this line was the only one that did not change

Comment (nashif) [reply CODEOWNERS]: or maybe yes
